include/geode/model/helpers/simplicial_brep_creator.h

```diff
@@ -55,9 +55,18 @@ namespace geode
         std::vector< uuid > create_surfaces( absl::Span< const uuid > lines,
             absl::Span< const SurfaceDefinition > definitions );
 
+        std::vector< uuid > create_surfaces( absl::Span< const uuid > corners,
+            absl::Span< const uuid > lines,
+            absl::Span< const SurfaceDefinition > definitions );
```

Comment:
Should we keep both? Maybe yes for now but look weird. We might need to rethink the class flow (someday)